runtime/wasmer_test.go

```diff
@@ -613,6 +646,59 @@ func TestExt_blake2_256_enumerated_trie_root(t *testing.T) {
 	}
 }
 
+// test that ext_twox_64 performs a xxHash64
+func TestExt_twox_64(t *testing.T) {
+	runtime, err := newTestRuntime()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	mem := runtime.vm.Memory.Data()
+	// save data in memory
+	// test for empty []byte
+	data := []byte(nil)
+	pos := 170
+	out := pos + len(data)
+	copy(mem[pos:pos+len(data)], data)
+
+	// call wasm function
+	testFunc, ok := runtime.vm.Exports["test_ext_twox_64"]
+	if !ok {
+		t.Fatal("could not find exported function")
+	}
+
+	_, err = testFunc(pos, len(data), out)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	//check result against expected value
+	if "99e9d85137db46ef" != hex.EncodeToString(mem[out:out+8]) {
```

Comment:
magical value

Reply:
i copied this from Ed's test... @edwardmack can you share where you got the twox test cases?

Reply:
These are from substrate code: https://github.com/paritytech/substrate/blob/13fc71c681cc9a3cc911c32c7890b52885092969/core/executor/src/wasm_executor.rs#L1701